src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,49 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core'
+import { IconButton, IconButtonSize } from '../../enums/icon-button'
+
+export interface MultiSelectOptions {
+  label: string
+  id: string
+  classesOnSelected?: string
+  classesOnList?: string
+}
+
+@Component({
+  selector: 'sofie-multi-select',
+  templateUrl: './multi-select.component.html',
+  styleUrls: ['./multi-select.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MultiSelectComponent {
+  @Input() public options: MultiSelectOptions[]
+  @Input() public selectedOptionsIds: string[]
+  @Input() public placeholder: string
+  @Input() public showOnlyItemsCountLabel?: string
```

Comment:
What does this variable do?
The name indicates that it is a boolean, but it is a string??